### DISCUSSION TITLE
Combine Border When Inner Gaps = 0

### DISCUSSION THREAD

**Discussion by ksolowoniuk** [hyprwm/Hyprland#5817] [General]
My personal config uses wide borders (8px). I think it would be nice if when setting 'gapps_in' to zero the borders between windows were combined so that the outer borders and the border between windows would be the same (8px).  Thanks

**Comment by vaxerski** [comment]
you can set negative gaps_in though?

**Reply by ksolowoniuk** [reply]
I will try when I get home and see how it works.

**Reply by ksolowoniuk** [reply]
It works. However I also use some transparency and using a negative value for gapps_in causes a shift in the level of transparency where the borders overlap. So it works but just not quite as cleanly as I had envisioned it. Thanks for the tip.

**Reply by vaxerski** [reply]
having it work cleanly would require a buncha more code and needless complexity

**Reply by ksolowoniuk** [reply]
No problem. You're doing nice work. I used to run VTWM and couldn't find a tiling wm that I liked til now.
